codespell_lib/data/dictionary_code.txt

```diff
@@ -21,6 +21,7 @@ seeked->sought
 sinc->sync, sink, since,
 sincs->syncs, sinks, since,
 stoll->still
+storaget->storage, storaged,
```

Comment:
`storaged` seems like a very odd suggestion? (but again perhaps this is because I don't understand the different dictionaries? :confused: )

Reply:
I found it as a valid word. I will kick it out, If you think it is not valid. 
The dictionary should not influence the suggestions (as far as I have understood it)

Reply:
Do you have a citation @sebweb3r ?

Reply:
https://en.wiktionary.org/wiki/storaged

Reply:
A proper citation, from a real dictionary? ( I'm just going to leave this here... https://www.theregister.com/2020/08/26/scots_wikipedia_fake/ )

Also:
```
storaged
    simple past tense and past participle of storage
```

Versus:
`store (<snip> simple past and past participle stored) `

I'd expect stored too, I can't think of a way to use storaged in a sentence).

Reply:
> The dictionary should not influence the suggestions (as far as I have understood it)

Also yes, you're correct, the multiple dictionaries are all about avoiding different classes of typos (e.g. real rare words, where the correction is more likely than the typo, or names or computing terms that would be typos outside those contexts)

Reply:
Thats why you are the gatekeeper :-)

I just checked, if there is an other solution possible. I wouldn't use storaged either, but I was not sure, if it is valid.
For english it is quite hard to find a reliable.
daemonize f.e. is also not in real dictionaries like https://dictionary.cambridge.org/spellcheck/english/?q=daemonize

Reply:
I've **never** heard "storage" used as a verb, only ever as a noun.

I think "daemonize" is a computing-specific term, derived from the "Unix daemon" sense.
It's probably used in a "making a process into a daemon (background task)" sense, but it's not _really_ a real word.

Reply:
I just referenced daemon due to this discussion https://github.com/codespell-project/codespell/pull/1653#issuecomment-683221197 :smile:

I'm fine with your suggestions. You are the native speakers.

I deleted storaged.

Reply:
Storaged has 424 hits on grep.app
https://grep.app/search?q=storaged&words=true
I would add this as "storaged->storage, stored," too, if you agree.

Reply:
Yeah, I'd guess in code-terms "storaged" is probably an abbreviation for "storage daemon", much like the way "maild" is an abbreviation for "mail daemon" (also ftpd, httpd, etc.).

Reply:
Yes, but there is a lot of people, who want to use stored too :-)